EKF/mag_fusion.cpp

```diff
@@ -623,14 +623,14 @@ void Ekf::fuseHeading()
 		}
 	}
 
-	// wrap the heading to the interval between +-pi
-	measured_hdg = wrap_pi(measured_hdg);
```

Comment:
I've looked at it as well and it does indeed seem to be the same. Strange. That would mean the only thing preventing convergence of the Kalman filter was the heading computation.